week-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function vat(num) {
```

Comment:
your function should take in a number, without that `£` sign. Can you come up with a better function name? eg `priceWithVat`, `calcuatePriceVat` etc